runtime/src/lib.rs

```diff
@@ -315,6 +315,14 @@ impl Contains<RuntimeCall> for FilteredCalls {
 	}
 }
 
+/// A type to identify allowed calls to the Runtime from contracts. Used by Pop API
+pub struct AllowedApiCalls;
+impl Contains<RuntimeCall> for crate::AllowedApiCalls {
+	fn contains(c: &RuntimeCall) -> bool {
+		matches!(c, RuntimeCall::Balances { .. } | RuntimeCall::Nfts { .. })
```

Comment:
I think we must be explicit with the individual calls here, a call should only be added if there is a corresponding implementation within the extension/api which requires it.